acs-engine.go

```diff
@@ -147,6 +155,22 @@ func main() {
 		os.Exit(1)
 	}
 
+	if (len(*caCertificatePath) > 0 && len(*caKeyPath) == 0) ||
```

Comment:
I don't quite understand.  The API model doesn't allow the specification of the CA private key.  The only way to supply the CA private key is through the flags.  How do you mean?